src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -269,7 +269,7 @@ SplitFullName(_In_z_ PCWSTR fullName,
 
         memberStart = memberEnd;
 
-        for (;;)
+        while (true)
```

Comment:
Is `while (true)` better than `for (;;)`? 

It fails my rule of thumb test that shorter is better when there are two equivalent ways to express something.

I do not have a strong opinion.

Reply:
`while (true)` is more readable, easier to format and is more frequently used in the repo than `for (;;)` and `while (1)` combined 682 vs. 147+56. IMO, we should streamline to a single style (preferably `while (true)`).